test/box-tap/suite.ini

```diff
@@ -4,7 +4,6 @@ description = Database tests with #! using TAP
 is_parallel = True
 use_unix_sockets_iproto = True
 release_disabled = errinj_set_with_enviroment_vars.test.lua
-disabled = gh-6198-max-cnt-of-tuple-fields-insert-overflow-err-msg.test.lua
```

Comment:
Since the test uses error injection, you should append it to the `release_disabled` list.

Reply:
Appended to the `release_disabled` list.

Reply:
Appended the test to the `release_disabled` list.